Scrumdinger/Views/DetailView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DetailView.swift
+//  Scrumdinger
+//
+//  Created by Life Boss on 7/16/23.
+//
+
+import SwiftUI
+
+struct DetailView: View {
+    let scrum: DailyScrum
+    
+    var body: some View {
+        List {
+            Section (header: Text("Meeting Info")) {
+                Label("Start Meeting", systemImage: "timer")
+                    .font(.headline)
+                    .foregroundColor(.accentColor)
```

Comment:
☝️ Note the use of `.accentColor` rather than `scrum.theme.accentColor`. This row is an interactive element, and by using the system `.accentColor` value this will render in the same color as other interactive elements.